api/api_full.go

```diff
@@ -864,9 +865,11 @@ type FullNode interface {
 	// IPC-specific methods //
 
 	// IPCAddSubnetActor deploys a new subnet actor.
-	IpcAddSubnetActor(ctx context.Context, wallet address.Address, params subnetactor.ConstructParams) (address.Address, error) //perm:write
-	IpcReadGatewayState(ctx context.Context, actor address.Address, tsk types.TipSetKey) (*gateway.State, error)                //perm:read
-	IpcReadSubnetActorState(ctx context.Context, actor address.Address, tsk types.TipSetKey) (*subnetactor.State, error)        //perm:read
+	IpcAddSubnetActor(ctx context.Context, wallet address.Address, params subnetactor.ConstructParams) (address.Address, error)   //perm:write
```

Comment:
`IpcSomething` must be `IPCSomething`.
Note that `// IPCAddSubnetActor deploys a new subnet actor.` is correct

Reply:
fixed